src/mono/wasm/debugger/DebuggerTestSuite/DebuggerTestSuite.csproj

```diff
@@ -98,4 +98,36 @@
                       Overwrite="true" />
   </Target>
 
+
+  <Target Name="CreateProjectWithColonInSourceName"
```

Comment:
Because we cannot have a file with colon in name on windows OS, only on linux/mac, if we push it to github when we try to checkout we get an error on windows.

Reply:
That reply can be converted to a comment:)